### ISSUE TITLE
The "popular repositories" on the `rust-bitcoin` org page are stale

### ISSUE THREAD

**Issue by tcharding** [rust-bitcoin/rust-bitcoin#2288]
Includes `rust-wallet` and `murmel`.

https://github.com/rust-bitcoin

**Comment by Kixunil** [comment]
That may be github-generated by number of forks?

**Comment by apoelstra** [comment]
Yeah, even from the admin page I don't see an obvious way to edit this list.

**Comment by Kixunil** [comment]
IIRC you can pin repositories but not sure if it'll hide the other ones.

**Comment by tcharding** [comment]
If its auto generated the algo is not good, `rust-bech32` has more forks and is more recently updated than `murmel` (34 vs 41).
